packages/sku/lib/configure.js

```diff
@@ -109,4 +113,24 @@ module.exports = async () => {
     comment: 'managed by sku',
     patterns: gitIgnorePatterns.map(convertToForwardSlashPaths),
   });
+
+  // Check for `.less` files
+  const lessFileGlobResults = await Promise.all(
+    paths.src
+      .filter((srcPath) => fs.statSync(srcPath).isDirectory())
+      .map(async (srcPath) => await glob(path.join(srcPath, '**/*.less'))),
+  );
+  const srcHasLessFiles = lessFileGlobResults.some((fileArray) => {
+    return fileArray.length > 0;
+  });
+  if (srcHasLessFiles) {
+    printBanner('warning', 'Less styles detected', [
+      `Vanilla Extract is the preferred styling solution supported by sku. Support for ${chalk.bold(
+        '.less',
+      )} styles may be removed in the future. Please migrate all ${chalk.bold(
+        '.less',
+      )} styles to Vanilla Extract styles.`,
```

Comment:
Thoughts on being a little firmer, but also leading with the deprecation call out?

> Support for LESS has been deprecated.
>
> Vanilla Extract is the preferred styling solution supported by sku, and support for LESS will be removed in a future release.
>
>  Consumers are encouraged to migrate to Vanilla Extract at their earliest opportunity.

Reply:
https://github.com/seek-oss/sku/pull/802/commits/5e5dc36ede97d4397c321f20c435a89fa6a7bec1